build/media-suite_compile.sh

```diff
@@ -161,14 +161,23 @@ if [[ $ripgrep = y || $rav1e = y || $dssim = y ]]; then
         add_to_remove
         cd_safe "$LOCALBUILDDIR"
     fi
-    if ! [[ $(rustup toolchain list) =~ stable-$CARCH-pc-windows-gnu ]]; then
+    if ! rustup toolchain list | grep -q "stable-$CARCH-pc-windows-gnu"; then
```

Comment:
There any special reason for this? The bash regex check is lighter on resources than using grep.

Reply:
Mainly for readability since it's easier to spot a grep than `=~`

I can revert if it's really unnecessary

Reply:
```suggestion
    if ! [[ $(rustup toolchain list) =~ stable-$CARCH-pc-windows-gnu ]]; then
```